src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SPECIAL_CHAR = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
+    private StringBuilder getFileStat(String fromFileName) {
+        File fileToRead = new File(fromFileName);
```

Comment:
you can use filename directly in `FileReader`  and `FileWriter` - no need to create `File` instances